tests/performance/README.md

```diff
@@ -1,31 +0,0 @@
-# Performance Testing
```

Comment:
Performance testing is used for load testing using locust.. Shall you keep this since is not related with your integration tests?

Reply:
`performance` directory is back now